presentation/notification/src/main/java/com/hilingual/presentation/notification/component/NotificationItem.kt

```diff
@@ -0,0 +1,122 @@
+package com.hilingual.presentation.notification.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.hilingual.core.common.extension.noRippleClickable
+import com.hilingual.core.designsystem.R
+import com.hilingual.core.designsystem.theme.HilingualTheme
+
+private val BoldNameRegex = Regex("(\\S*님\\S*)")
+
+@Composable
+internal fun NotificationItem(
+    title: String,
+    date: String,
+    isRead: Boolean,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    val typo = HilingualTheme.typography
+    val colors = HilingualTheme.colors
+    val backgroundColor = if (isRead) colors.white else colors.gray100
+
+    Row(
+        modifier = modifier
+            .noRippleClickable(onClick = onClick)
+            .fillMaxWidth()
+            .border(1.dp, colors.gray100)
+            .background(backgroundColor)
+            .padding(vertical = 20.dp)
+            .padding(start = 16.dp, end = 12.dp),
+        horizontalArrangement = Arrangement.spacedBy(4.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Column(
+            modifier = Modifier.weight(1f),
+            verticalArrangement = Arrangement.spacedBy(4.dp)
+        ) {
+            Text(
+                text = buildAnnotatedString {
+                    val matchResult = BoldNameRegex.find(title)
+                    if (matchResult != null) {
+                        val range = matchResult.range
+                        append(title.substring(0, range.first))
+                        withStyle(style = SpanStyle(fontWeight = FontWeight.Bold)) {
+                            append(matchResult.value)
+                        }
+                        append(title.substring(range.last + 1))
```

Comment:
p3
중간에만 텍스트를 바꾸는 거라면 `withStyle`말고 아래처럼 `addStyle`을 사용하는 방법도 있을 것 같아요!
```suggestion
                        val range = matchResult.range
                        append(title)
                        addStyle(
                            style = SpanStyle(fontWeight = FontWeight.Bold)) {
                                append(matchResult.value)
                            }
                            start = range.first,
                            end = range.last
                        )
        }
```

Reply:
좋네요! 적용하겠습니다 :)